src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -735,7 +758,7 @@ private async Task DispatchInvocationAsync(InvocationMessage invocation)
             {
                 try
                 {
-                    await handler.InvokeAsync(invocation.Arguments);
+                    handler.InvokeAsync(invocation.Arguments);
```

Comment:
I like not blocking the handlers on the async execution of the previous handler, but I still think the handler should be wrapped in something that awaits it so async exceptions can be logged.

Reply:
Never mind, just go back to awaiting.

Reply:
👍